### ISSUE TITLE
Parts Request: Large Arcade Button

### ISSUE THREAD

**Issue by sekjal** [adafruit/Fritzing-Library#63]
I would find it very useful to have the following parts available:

- https://www.adafruit.com/products/1190
- https://www.adafruit.com/products/1191
- https://www.adafruit.com/products/1192
- https://www.adafruit.com/products/1193
- https://www.adafruit.com/products/1194

This would allow me to more clearly communicate the wiring of my project, where both the button circuit and the LED circuit are connected to the same physical part (rather than showing separate buttons and LEDs).

Thank you for considering.

**Comment by Ramblurr** [comment]
Yes please! Also the ["Massive"](https://www.adafruit.com/product/1187) and ["Mini"](https://www.adafruit.com/product/3429) buttons too.
